python/Ganga/GPIDev/Adapters/IPrepareApp.py

```diff
@@ -45,7 +53,8 @@ def prepare(self, force=False):
         """
         Base class for all applications which can be placed into a prepared\
         state. 
-
+        Args:
+            force (bool) : ignored by the base class. Higher up forces the prepare function to be called no matter what
```

Comment:
There's no cheat sheet, just standard 'rules' for code documentation, describe the interface rather than implementation etc.